sound/soc/sof/core.c

```diff
@@ -220,15 +220,15 @@ EXPORT_SYMBOL(snd_sof_get_status);
  * into compressed page table.
```

Comment:
@ranj063 if there is no trouble, I would really like you to state in the commit message WHY is this change needed.